scripts/gtcadis.py

```diff
@@ -122,19 +139,60 @@ def _cards(source):
              }
     return cards
 
-
-def step1(cfg):
-    """ This function writes the PARTISN input file for the adjoint photon 
-    transport   
+def step0(cfg1, cfg2, clean):
+    """
+    This function performs the SNILB criteria check.
+    
     Parameters
     ----------
-    cfg : dictionary
+    cfg1 : dictionary
+        User input for Step 1 from the config.yml file
+    cfg2 : dictionary
+        User input for Step 2 from the config.yml file    
+    clean: str
+        User input for condition on deleting the intermediate files
+    """
+    # Get user input from config file
+    geom = cfg2['geom_file']
+    data_dir = cfg2['data_dir']
+    irr_times = str(cfg2['irr_time']).split(' ')
+    decay_times = str(cfg2['decay_time']).split(' ')
+    num_p_groups = cfg1['p_groups']
+    
+    # Define a flat, 175 group neutron spectrum, with magnitude 1E12 [n/s]
+    neutron_spectrum = [1]*175  # will be normalized
+    flux_magnitudes = [1.75E14] # 1E12*175
```

Comment:
Is this the mag of the total flux?

Reply:
yes. the spectrum is normalized by the total (175) and in alara input we will have a 1.75e14 flux multiplication factor so this way the group fluxes will be 1e12

Reply:
with the above change, this could be
```
group_flux_magnitude = 1e12
flux_magnitude = [n_groups * group_flux_magnitude]
```